server/index.js

```diff
@@ -1,23 +1,31 @@
 const express = require('express');
-const app = express();
+const server = express();
 const bodyParser = require('body-parser');
 const db = require('../database/index');
+const {getReservations} = require('../database/index');
 
 const port = 3003;
 
-app.use(express.static('./public/dist'));
-app.use(bodyParser.json());
+server.use(express.urlencoded({ extended: true }));
+server.use(express.static('./public/dist'));
+server.use(bodyParser.json());
 
-// app.get('/calendar', function(req, res){
-//     console.log(req.body);
-//     res.send('hello world');
-// });
-
-app.post('/calendar', (req, res) => {
-    console.log("INITIALIZED POST, GOING TO DB...");
+server.post('/calendar', (req, res) => {
     db.addCalendar(req.body);
     res.send();
```

Comment:
should res.send() or something else be used to confirm successful post?

Reply:
I gutted out and refactored my whole front-end AND back-end today so it's not using local dummy data on my front-end anymore. It's now actually getting the data that was seeded into the db using a now self-contained seeding script (thanks for helping me with that).